dashboard/config/scripts/levels/Dance Party 1-validated.level

```diff
@@ -55,13 +55,18 @@
unc\": \"wander\",\r\n    \"blockText\": \"wandering\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"grow\",\r\n    \"blockText\": \"growing\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"shrink\",\r\n    \"blockText\": \"shrinking\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"spin\",\r\n    \"blockText\": \"spinning\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"danceParty\",\r\n    \"blockText\": \"dancing\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"runUp\",\r\n    \"blockText\": \"running north\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"runDown\",\r\n    \"blockText\": \"running south\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"runLeft\",\r\n    \"blockText\": \"running west\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"runRight\",\r\n    \"blockText\": \"running east\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"patrollingLeftRight\",\r\n    \"blockText\": \"patrolling left and right\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"faceDirection\",\r\n    \"blockText\": \"{SPRITE} face {DIRECTION}\",\r\n    \"args\": [\r\n      {\r\n        \"name\": \"SPRITE\",\r\n        \"type\": \"Sprite\"\r\n      },\r\n      {\r\n        \"name\": \"DIRECTION\",\r\n        \"options\": [\r\n          [\r\n            \"right\",\r\n            \"1\"\r\n          ],\r\n          [\r\n            \"left\",\r\n            \"-1\"\r\n          ]\r\n        ]\r\n      }\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"followingSprite\",\r\n    \"blockText\": \"following {TARGET}\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ],\r\n    \"args\": [\r\n      {\r\n        \"name\": \"TARGET\",\r\n        \"type\": \"Sprite\"\r\n      }\r\n    ]\r\n  },\r\n  {\r\n    \"func\": \"wanderRewrite\",\r\n    \"blockText\": \"wandering (new)\",\r\n    \"returnType\": \"Behavior\",\r\n    \"color\": [\r\n      136,\r\n      \".84\",\r\n      \".80\"\r\n    ]\r\n  }\r\n]",
```

Comment:
It's a little hard to review from this diff but I'm assuming you just deleted these two custom blocks out of this level?
What are the rest of the custom blocks here? Can we delete them too?

Reply:
I believe so. I'll delete them and double check